### DISCUSSION TITLE
** (ArithmeticError) bad argument in arithmetic expression

### DISCUSSION THREAD

**Discussion by kumfo** [plausible/analytics#1714] [Self-Hosted Support] [answered]
when I request the API path to stats goal, it response error:

* API path: `/api/stats/:site/pages`
* payload: 
```
period: 30d
date: 2022-02-25
filters: {"goal":"viewed","props":{"user_id":"1111"}}
```

* response:
```json
{"error":"** (ArithmeticError) bad argument in arithmetic expression"}
```

deploy with docker by latest tag

Thanks

**Comment by kumfo** [comment, marked as answer]
Originally, I called it from the server and only stored the custom goal.

When I save the custom goal, and save "pageview" event also, I found it work.

But I don't know what it's happen.

Thanks.

**Comment by adamekberg** [comment]
Hey! I am dealing with this issue also. I'm not sure I understand what you did to solve it. Do you mind explaining?
